src/FormDemo.vue

```diff
@@ -102,6 +102,31 @@
             }
           ]
         }
+      },
+      {
+        type: 'number',
```

Comment:
I would be nice both for development and documentation to use a molgenis entity and use the mapper function, while merging the mapper code into the master a across several issues that would not have been caught the mapper was not used.